projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -164,6 +176,22 @@ export class TableWidgetRendererComponent
     );
   }
 
+  private applySavedColumnPreferences(
+    columns: SpecificationBackedTableColumnDef[],
+    persistedColumns: TableColumnConfig[]
+  ): SpecificationBackedTableColumnDef[] {
+    return columns.map(column => {
+      const found = persistedColumns.find(persistedColumn => persistedColumn.id === column.id);
+
+      return {
+        ...column, // Apply default column config
+        ...(found ? found : {}), // Override with any saved properties
+        specification: column.specification, // Spec not user configurable and has nested methods, so apply those
```

Comment:
Can we change it so we only persist the specific settings worth persisting - it feels like going default, persisted, some defaults is an easy way to introduce bugs if a new method for example is added to the col.